src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -989,7 +989,7 @@ namespace MyApp.Pages
     using global::System.Threading.Tasks;
     using global::Microsoft.AspNetCore.Components;
 #nullable restore
-#line 2 ""Pages/Index.razor""
+#line (2,2)-(3,1) 1 ""Pages/Index.razor""
 using SurveyPromptRootNamspace;
```

Comment:
Do we have a test where we have a space/newline before the explicit semicolon? And where we have multiple using statements on one line?

```cs
using System ;using System
; using System;
```

Reply:
Added a test to cover this. Interestingly the parsing treats `@using namespace[ ];` as being a using statement, with a markup literal `;` which is weird, but we at least map what was parsed correctly.